### PR TITLE
add function to check for colons in filenames

### PR DESCRIPTION
**Pull request by privatezero**

### CODE REVIEW

**Comment by dericed** [comment]
:

**Comment by privatezero** [comment]
hahaha I am gonna assume that means looks ok?

File: mmfunctions [hunk 1]
```diff
@@ -20,6 +20,16 @@ _cleanup_mount_dir(){
     fi
 }
 
+_check_for_colons(){
+    COLONALERT=$(find "${1}" -iname '*:*')
+    if [ -n "${COLONALERT}" ] ; then
+        _report -w "Illegal characters (colons) have been detected in the following file(s)"
+        _report -w "${COLONALERT}"
```
Comment (retokromer) [inline mmfunctions]: For readability add `:` at line 26 and indent list at line 27?
```
        _report -w "Illegal characters (colons) have been detected in the following file(s):"
        _report -w "  ${COLONALERT}"
```

**Comment by retokromer** [comment]
LGTM

**Pull request merged**